.github/workflows/unroute-new-issue.yml

```
@@ -0,0 +1,33 @@
+name: Unroute new issue
```

"Unroute" sounds like you are undoing something, how about this
```suggestion
name: Mark new issue for routing
```

---

https://github.com/getsentry/sentry/pull/26964#issuecomment-870913818